<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Form</class>
    <widget class="QWidget" name="Form">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>900</width>
                <height>700</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form Pengembalian Peralatan</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QGroupBox" name="groupBox_Atas">
                    <property name="title">
                        <string>Data Transaksi Pengembalian</string>
                    </property>
                    <layout class="QGridLayout" name="gridLayout">
                        <item row="0" column="0">
                            <widget class="QLabel" name="label_3">
                                <property name="text">
                                    <string>Pilih ID Penyewaan:</string>
                                </property>
                            </widget>
                        </item>
                        <item row="0" column="1" colspan="3">
                            <widget class="QComboBox" name="comboBox_IdSewa">
                                <property name="toolTip">
                                    <string>Pilih ID Penyewaan yang aktif</string>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="0">
                            <widget class="QLabel" name="label_2">
                                <property name="text">
                                    <string>Tanggal Kembali (Aktual):</string>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="1">
                            <widget class="QDateEdit" name="dateEdit_TglKembaliAktual">
                                <property name="displayFormat">
                                    <string>yyyy-MM-dd</string>
                                </property>
                                <property name="calendarPopup">
                                    <bool>true</bool>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="2">
                            <widget class="QLabel" name="label">
                                <property name="text">
                                    <string>No Pengembalian (Auto):</string>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="3">
                            <widget class="QLineEdit" name="lineEdit_NoPengembalian">
                                <property name="placeholderText">
                                    <string>Otomatis</string>
                                </property>
                                <property name="readOnly">
                                    <bool>true</bool>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <widget class="QGroupBox" name="groupBox_Pelanggan">
                    <property name="title">
                        <string>Informasi Pelanggan &amp; Proyek (Otomatis Terisi)</string>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_2">
                        <item row="0" column="0">
                            <widget class="QLabel" name="label_4">
                                <property name="text">
                                    <string>ID Pelanggan:</string>
                                </property>
                            </widget>
                        </item>
                        <item row="0" column="1">
                            <widget class="QLineEdit" name="lineEdit_IdPel">
                                <property name="readOnly">
                                    <bool>true</bool>
                                </property>
                                <property name="placeholderText">
                                    <string>Otomatis...</string>
                                </property>
                            </widget>
                        </item>
                        <item row="0" column="2">
                            <widget class="QLabel" name="label_5">
                                <property name="text">
                                    <string>Nama Perusahaan:</string>
                                </property>
                            </widget>
                        </item>
                        <item row="0" column="3">
                            <widget class="QLineEdit" name="lineEdit_NmPerusahaan">
                                <property name="readOnly">
                                    <bool>true</bool>
                                </property>
                                <property name="placeholderText">
                                    <string>Otomatis...</string>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="0">
                            <widget class="QLabel" name="label_6">
                                <property name="text">
                                    <string>Lokasi Proyek:</string>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="1" colspan="3">
                            <widget class="QLineEdit" name="lineEdit_Lokasi">
                                <property name="readOnly">
                                    <bool>true</bool>
                                </property>
                                <property name="placeholderText">
                                    <string>Otomatis...</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <widget class="QGroupBox" name="groupBox_Alat">
                    <property name="title">
                        <string>Informasi Alat Berat &amp; Operator (Otomatis Terisi)</string>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_3">
                        <item row="0" column="0">
                            <widget class="QLabel" name="label_7">
                                <property name="text">
                                    <string>ID Alat:</string>
                                </property>
                            </widget>
                        </item>
                        <item row="0" column="1">
                            <widget class="QLineEdit" name="lineEdit_IdAlat">
                                <property name="readOnly">
                                    <bool>true</bool>
                                </property>
                                <property name="placeholderText">
                                    <string>Otomatis...</string>
                                </property>
                            </widget>
                        </item>
                        <item row="0" column="2">
                            <widget class="QLabel" name="label_8">
                                <property name="text">
                                    <string>Nama Alat:</string>
                                </property>
                            </widget>
                        </item>
                        <item row="0" column="3">
                            <widget class="QLineEdit" name="lineEdit_NamaAlat">
                                <property name="readOnly">
                                    <bool>true</bool>
                                </property>
                                <property name="placeholderText">
                                    <string>Otomatis...</string>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="0">
                            <widget class="QLabel" name="label_9">
                                <property name="text">
                                    <string>ID Operator:</string>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="1">
                            <widget class="QLineEdit" name="lineEdit_IdOp">
                                <property name="readOnly">
                                    <bool>true</bool>
                                </property>
                                <property name="placeholderText">
                                    <string>Otomatis...</string>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="2">
                            <widget class="QLabel" name="label_10">
                                <property name="text">
                                    <string>Nama Operator:</string>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="3">
                            <widget class="QLineEdit" name="lineEdit_NamaOp">
                                <property name="readOnly">
                                    <bool>true</bool>
                                </property>
                                <property name="placeholderText">
                                    <string>Otomatis...</string>
                                </property>
                            </widget>
                        </item>
                        <item row="2" column="0">
                            <widget class="QLabel" name="label_11">
                                <property name="text">
                                    <string>Tgl Sewa Awal:</string>
                                </property>
                            </widget>
                        </item>
                        <item row="2" column="1">
                            <widget class="QLineEdit" name="lineEdit_TglSewa">
                                <property name="readOnly">
                                    <bool>true</bool>
                                </property>
                                <property name="placeholderText">
                                    <string>Otomatis...</string>
                                </property>
                            </widget>
                        </item>
                        <item row="2" column="2">
                            <widget class="QLabel" name="label_12">
                                <property name="text">
                                    <string>Est. Kembali:</string>
                                </property>
                            </widget>
                        </item>
                        <item row="2" column="3">
                            <widget class="QLineEdit" name="lineEdit_EstKembali">
                                <property name="readOnly">
                                    <bool>true</bool>
                                </property>
                                <property name="placeholderText">
                                    <string>Otomatis...</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <widget class="QGroupBox" name="groupBox_4">
                    <property name="title">
                        <string>Catatan &amp; Denda</string>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_4">
                        <item row="0" column="0">
                            <widget class="QLabel" name="label_13">
                                <property name="text">
                                    <string>Keterangan / Kondisi Alat:</string>
                                </property>
                            </widget>
                        </item>
                        <item row="0" column="1">
                            <widget class="QLineEdit" name="lineEdit_Keterangan">
                                <property name="placeholderText">
                                    <string>Contoh: Baik, Rusak Ringan, Kotor, dll...</string>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="0">
                            <widget class="QLabel" name="label_14">
                                <property name="text">
                                    <string>Denda (Jika ada):</string>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="1">
                            <widget class="QLineEdit" name="lineEdit_Denda">
                                <property name="text">
                                    <string>0</string>
                                </property>
                                <property name="placeholderText">
                                    <string>Input angka denda jika ada keterlambatan/kerusakan</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                        <widget class="QPushButton" name="pushButton_Baru">
                            <property name="text">
                                <string>Bersihkan Form</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="pushButton_Simpan">
                            <property name="text">
                                <string>Proses Pengembalian</string>
                            </property>
                            <property name="styleSheet">
                                <string notr="true">background-color: #4CAF50; color: white; font-weight: bold;</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <spacer name="horizontalSpacer">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <widget class="QPushButton" name="pushButton_Keluar">
                            <property name="text">
                                <string>Keluar</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="QTableWidget" name="tableWidget">
                    <column>
                        <property name="text">
                            <string>ID Pengembalian</string>
                        </property>
                    </column>
                    <column>
                        <property name="text">
                            <string>Tgl Kembali</string>
                        </property>
                    </column>
                    <column>
                        <property name="text">
                            <string>ID Sewa</string>
                        </property>
                    </column>
                    <column>
                        <property name="text">
                            <string>Kondisi</string>
                        </property>
                    </column>
                    <column>
                        <property name="text">
                            <string>Denda</string>
                        </property>
                    </column>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>